android/app/src/main/java/com/microsoft/codepush/react/CodePush.java

```diff
@@ -299,11 +312,10 @@ public static void setUsingTestConfiguration(boolean shouldUseTestConfiguration)
         testConfigurationFlag = shouldUseTestConfiguration;
     }
 
-    public void clearTestUpdates() {
-        if (isUsingTestConfiguration()) {
-            codePushPackage.clearTestUpdates();
-            removePendingUpdate();
-        }
+    public void clearUpdates() {
+        codePushPackage.clearUpdates();
```

Comment:
I just worry about getting an update available over and over again in dev mode when you're not bumping the version but you're redeploying new binaries. The update check wouldn't send a hash (since it was cleared) so the server would keep giving it the update. In iOS, that wouldn't happen due to the behavior we discussed, but on Android it seems like this could happen.